database/sqlite/sqlite_functions.c

```diff
@@ -697,20 +697,21 @@ void delete_dimension_uuid(uuid_t *dimension_uuid)
  * Do a database lookup to find the UUID of a chart
  *
  */
-uuid_t *find_chart_uuid(RRDHOST *host, const char *type, const char *id, const char *name)
+int find_chart_uuid(RRDHOST *host, const char *type, const char *id, const char *name, uuid_t *store_uuid)
 {
     static __thread sqlite3_stmt *res = NULL;
     uuid_t *uuid = NULL;
```

Comment:
I am starting the review tonight, and I observed when I compiled this branch I am receiving this warning:

```sh
database/sqlite/sqlite_functions.c: In function 'find_chart_uuid':
database/sqlite/sqlite_functions.c:703:13: warning: unused variable 'uuid' [-Wunused-variable]
  703 |     uuid_t *uuid = NULL;
      | 
```

Reply:
@stelfrag this is yours...